doc/dds-core-xtypes/README.md

```diff
@@ -101,15 +103,15 @@ my_struct.add_member("m_e", other_struct)); //member of structs
 my_struct.add_member("m_f", SequenceType(other_struct))); //member of sequence of structs
 ```
 Note: once a `DynamicType` is added to an struct, a copy is performed.
-This allow to modify the `DynamicType`s without side effects, and facilitate the memory management by the user.
+This allows modifications to `DynamicType` to be performed without side effects. It also and facilitates the user's memory management duties.
 
 #### `is_compatible` function of `DynamicType`
-Any `DynamicType` can be compared with another `DynamicType` to check its compatibility.
+Any pair of `DynamicType`s can be checked for their mutual compatibility (note that A's compatibility with B does not imply that B is therefore compatible with A).
```

Comment:
The last part of the sentence is wrong, the idea now is that the compatibility is in both directions.